packages/types/next-auth.d.ts

```diff
@@ -16,6 +16,7 @@ declare module "next-auth" {
     email_verified?: boolean;
     impersonatedByUID?: number;
     belongsToActiveTeam?: boolean;
+    organizationId?: number | null;
```

Comment:
This was removed but still was being used throughout the code, so bringing it back officially.